script/Arcueid/Day 7/07_00_ARC07_1D.txt

```diff
@@ -226,10 +240,10 @@ The night has just begun. I should―――
 [sha:8029f80b169796c4745bc555a441363c03623a78]{
 -- Page 130, Offset 42635. Choice.
 -- いったん屋敷に戻って整理しよう。
-Return to the mansion and reassess the situation.
+ Return to the mansion and reassess the situation.
 }
 [sha:056d9afee7598989cc23923812a064a2c5588f2a]{
 -- Page 130, Offset 42636. Choice.
 -- いや、諦めずに街を回ろう。
-No, I should keep searching the city.
+ Not give up, and keep searching the city.
```

Comment:
I thought this was choice dialogue or something. Did you check in game?

Reply:
It is a choice, which is why I've worded it so that the leading ask matches the answer:

> The night has just begun. I should――― Return to the mansion and reassess the situation.
> The night has just begun. I should――― Not give up, and keep searching the city.